express-hello-world/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+
+app.use(function (req, res) {
+  console.log('method:', req.method());
+  String(res.send());
```

Comment:
No need to use the `String` constructor here. We just want to send back a sting:
```JAVASCRIPT
res.send('Hello, express');
```